cmd/cortex/main.go

```diff
@@ -60,6 +66,7 @@ func main() {
 		level.Error(util.Logger).Log("msg", "error running Cortex", "err", err)
 	}
 
+	runtime.KeepAlive(ballast)
```

Comment:
Aha! I tried doing this in a different project awhile ago and did not know about KeepAlive.